assets/js/modules/sign-in-with-google/components/settings/SettingsEdit.stories.js

```diff
@@ -72,12 +89,12 @@ Default.decorators = [
 
 export const InvalidClientID = Template.bind( null );
 InvalidClientID.storyName = 'Invalid Client ID';
-InvalidClientID.scenario = {
-	label: 'Modules/Sign in with Google/Settings/SettingsForm/Invalid Client ID',
-};
+InvalidClientID.scenario = {};
```

Comment:
We don't need a SiwG story to cover a standard input field error; this should be removed.